src/main/webapp/views/editCard.html

```diff
@@ -15,104 +15,109 @@
-                  <input type="text" class="form-control" id="business-name" aria-describedby="Business Name"
-                    placeholder="Business Name">
+                  <label for="businessName"">Business Name:</label>
+                  <input type="text" class="form-control" id="businessName" name="businessName"
+                         aria-describedby="Business Name"
+                         placeholder="Business Name">
                 </div>
               </div>
               <div class="row mt-4">
                 <div class="col">
-                  <label for="business-description">Business description:</label>
-                  <textarea class="form-control" id="business-description" rows="3"
-                    placeholder="Your business description"></textarea>
+                  <label for="description">Business description:</label>
+                  <textarea class="form-control" id="description" rows="3" name="description"
+                            placeholder="Your business description"></textarea>
                 </div>
               </div>
               <hr>
               <div class="row mt-4">
                 <div class="col-lg-3">
                   <h5>Current business photo</h5>
                   <img id="business-picture" src="images/placeholderImage.png" class="card-img-top mt-2"
-                    alt="images/placeholderImage.png">
-                  <label class="mt-2" for="business-photo"><b>Select a new business photo</b></label>
-                  <input type="file" class="form-control-file" id="business-photo">
+                       alt="images/placeholderImage.png">
+                  <label class="mt-2" for="imageFile"><b>Select a new business photo</b></label>
+                  <input type="hidden" id="blobKey" name="blobKey">
+                  <input type="file" class="form-control-file" id="imageFile" name="imageFile">
                 </div>
                 <div class="col">
                   <p>Business Working Hours</p>
                   <div class="row">
                     <div class="col">
                       <p>From</p>
-                      <input type="time" class="form-control" id="business-startTime"
-                        aria-describedby="Business opened from" placeholder="From">
+                      <input type="time" class="form-control" id="startTime" name="startTime"
+                             aria-describedby="Business opened from" placeholder="From">
                     </div>
                     <div class="col">
                       <p>To</p>
-                      <input type="time" class="form-control" id="business-endTime"
-                        aria-describedby="Business opened to" placeholder="To">
+                      <input type="time" class="form-control" id="endTime" name="endTime"
+                             aria-describedby="Business opened to" placeholder="To">
```

Comment:
The value of `aria-describedby` needs to be an id.
[More info about it here.](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute)